src/CommonITILCost.php

```diff
@@ -438,8 +438,17 @@ public function showForm($ID, array $options = [])
         echo "<tr class='tab_bg_1'>";
         echo "<td>" . __('Duration') . "</td>";
         echo "<td>";
+
+        $toadd = [];
+        $max = max(DAY_TIMESTAMP, 10 * MINUTE_TIMESTAMP);
```

Comment:
This part seems odd, `showTimestamp` method doesn't support it directly ? Or maybe we can add some new feature to it, no ?